src/fileseq/filesequence.py

```diff
@@ -450,19 +457,19 @@ def yield_sequences_in_list(paths):
             if frame:
                 seqs[key].add(frame)
 
-        for (dirname, basename, ext), frames in seqs.iteritems():
+        for (dirname, basename, ext), frames in seqs.items():
```

Comment:
Since six is now a dependency of the project, would it be better to use `six.viewitems` or `six.iteritems` instead of `items()` so that py2 dodesn't have to take the hit on efficiency?

Reply:
Yeah sure man - I can change that.